mobile/ios/device/ios-application-manager.ts

```diff
@@ -92,8 +93,13 @@ export class IOSApplicationManager extends ApplicationManagerBase {
 	}
 
 	public async restartApplication(applicationId: string, appName?: string): Promise<void> {
-		await this.stopApplication(applicationId, appName);
-		await this.runApplicationCore(applicationId);
+		try {
+			await this.stopApplication(applicationId, appName);
+			await this.runApplicationCore(applicationId);
+		} catch (err) {
+			await this.$iOSNotificationService.postNotification(this.device.deviceInfo.identifier, `${applicationId}:NativeScript.LiveSync.RestartApplication`);
```

Comment:
so... in case we are restarting Cordova App ?
Or NativeScript app built with Telerik Platform?